### PR TITLE
Add 20240410 minutes

### PR DESCRIPTION
**Pull request by planetf1**
Adds an initial version of the minutes from 2024-04-10

@ryjones @dstebila can you point me to/share the meeting recording/transcript & I'll update with any omissions.

### CODE REVIEW

**Comment by dstebila** [comment]
Thanks Nigel! Here's the recording & transcript: https://zoom.us/rec/play/SSS-KDTuL7-MfCQaBoOH6NsJ2qRjPuc46X1hzaRizeJBeFsgRlTapONnlosGDbBGjAJI8dksmehEiF0I.s2Hw9yHdlBvJ0s-A?canPlayFromShare=true&from=share_recording_detail&continueMode=true&componentName=rec-play&originRequestUrl=https%3A%2F%2Fzoom.us%2Frec%2Fshare%2FoQXCKON8wrHgBsjAWgWQYZxOBfKB9eeiyw38CgqMbhVB9LqHnE0cPMhwdW3zZDYp.ID4HoVpw_jj0841b

**Comment by planetf1** [comment]
Thanks. I'll do the next iteration in next day.

**Comment by planetf1** [comment]
I've made a few small changes, including adding the recording
Moving out of draft - these are ready for review now.

**Comment by planetf1** [comment]
Do we merge when the TSC approves in the next call, or merge before that, with the approval still occuring in the meeting?

**Comment by ryjones** [comment]
@planetf1 I would think the TSC members mark it as approved, and merge once you reach the level required?

**Review by dstebila — APPROVED** [review]

**Review by thb-sb — APPROVED** [review]

**Review by bhess — APPROVED** [review]

**Comment by baentsch** [comment]
> @planetf1 I would think the TSC members mark it as approved, and merge once you reach the level required?

@ryjones @planetf1 the "OQS culture" so far was that PRs don't get merged with  questions at least given feedback.

**Review by baentsch — COMMENTED** [review]
I asked for feedback/clarifications (whether I understood things right) on 3 line items.

**Review by ashman-p — APPROVED** [review]

File: meetings/2024-04-10/minutes.md
Comment (dstebila) [reply meetings/2024-04-10/minutes.md:90]: That seems like a separate discussion to the minutes of the OQS-BoringSSL license, unless I'm missing the connection?

File: meetings/2024-04-10/minutes.md
Comment (dstebila) [reply meetings/2024-04-10/minutes.md:74]: Are you proposing to change the word "May" to "Will"?

File: meetings/2024-04-10/minutes.md
Comment (baentsch) [reply meetings/2024-04-10/minutes.md:90]: Well, that was the context in which IBM/LF (think MM, but am not sure) made the remark. As I did not find another place in the minutes in which this was captured, I commented here.

File: meetings/2024-04-10/minutes.md
Comment (baentsch) [reply meetings/2024-04-10/minutes.md:74]: "must" or "needs to" seems to be more suitable than "will" given the original word was "may". Anyway, the core of the matter is: What designations should be applied to which projects or components is unclear to me.

**Review by vsoftco — APPROVED** [review]

**Comment by planetf1** [comment]
> > @planetf1 I would think the TSC members mark it as approved, and merge once you reach the level required?
> 
> @ryjones @planetf1 the "OQS culture" so far was that PRs don't get merged with questions at least given feedback.

In previous projects we've enforced that conversations are closed before merging - agree.

File: meetings/2024-04-10/minutes.md
Comment (planetf1) [reply meetings/2024-04-10/minutes.md:74]: I think there's a few issues here
 - As minutes, these should reflect the discussion. If things are missed out, incorrect, don't adequately capture the discussion then absolutely we should update. Since there wasn't a firm decision I felt that using the word 'may' was appropriate in the minutes, but if I misrepresented, can you suggest alternate words?
 - The PQCA are delegating the creation of the lifecycle document to oqs as I understand it. We could take a copy of the initial draft, and collaboratively review/update through google docs etc, or via a github PR - whichever works best. Do you want to own it @baentsch ? @dstebila If no-one else does I can (but the reason it's come back to oqs is that you have the most experience and skin in the game..) , I think it's important. I think this is part of https://github.com/open-quantum-safe/tsc/issues/1 - ie it's more about policy & is fed back upstream to the pqca
  - https://github.com/open-quantum-safe/tsc/issues/2 is more a result of applying the policy - mapping to existing repos/capabilities

File: meetings/2024-04-10/minutes.md
Comment (planetf1) [reply meetings/2024-04-10/minutes.md:78]: I think we start with a top level issue at the tsc level since it applies to the whole project (gh org).
I was going to raise an issue, but we have https://github.com/open-quantum-safe/tsc/issues/1 open already - are you ok we use that. Can update minutes with the link?

File: meetings/2024-04-10/minutes.md
Comment (planetf1) [reply meetings/2024-04-10/minutes.md:90]: There was a discussion right at the end of the meeting where @baentsch asked if it's true we can't fork projects that aren't Apache/MIT. Both Max and Hart responded that generally would recommend no/could be tricky. I don't believe it was an 'ever' closed statement.

The charter does state MIT only currently - the referenced issue has more comments.

So I think the meetings notes are representative. The full transcript can be checked [here]
(https://zoom.us/rec/play/SSS-KDTuL7-MfCQaBoOH6NsJ2qRjPuc46X1hzaRizeJBeFsgRlTapONnlosGDbBGjAJI8dksmehEiF0I.s2Hw9yHdlBvJ0s-A?canPlayFromShare=true&from=share_recording_detail&continueMode=true&componentName=rec-play&originRequestUrl=https%3A%2F%2Fzoom.us%2Frec%2Fshare%2FoQXCKON8wrHgBsjAWgWQYZxOBfKB9eeiyw38CgqMbhVB9LqHnE0cPMhwdW3zZDYp.ID4HoVpw_jj0841b) - I presume they will expire at some point

**Review by baentsch — APPROVED** [review]
Thanks for the feedback @planetf1 . Conversations marked as resolved.

**Pull request merged**
